src/win32structs.h

```diff
@@ -83,15 +83,21 @@
 #define MWT_LEFTMULTIPLY	2
 #define MWT_RIGHTMULTIPLY	3
 
+typedef guint32 ARGB;
+typedef float REAL;
+
+#if defined(_WIN32)
+#define NOGDI
+#include <Windows.h>
+#else
+
 typedef int LANGID;
```

Comment:
unrelated to the PR but according to https://msdn.microsoft.com/en-us/library/windows/desktop/aa383751(v=vs.85).aspx LANGID should be WORD (16-bit unsigned integer), defining it as int looks like a bug to me?